### DISCUSSION TITLE
test defects: how to remove

### DISCUSSION THREAD

**Discussion by leder11011** [allure-framework/allure2#2660] [Questions & Support] [answered]
Hello all,

thank you for allure2: I have version 2.30.0 up and running on manjaro linux.
In the generated report `allure generate --clean` there exists a test labeled `broken`. I have already removed the test in the corresponding feature file: How to remove the test execution from the allure report?

**Comment by delatrie** [comment, marked as answer]
Hi, @leder11011 !

Clean the results directory (the one where `.json` files are saved; typically, `allure-results`). Then rerun the tests and create a new report.
